backend/src/main/scala/fetch_managers/AlbanyCourseFetchManager.scala

```diff
@@ -0,0 +1,261 @@
+package fetch_managers
+
+import com.typesafe.scalalogging.slf4j.LazyLogging
+import course_refresh.{HTTPRequest, HTTPManager, SchoolManager}
+import model._
+
+import scala.concurrent.Future
+import scala.xml.{Node, NodeSeq}
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class AlbanyCourseFetchManager(term: String) extends SchoolManager with LazyLogging {
+  lazy val allDepartments: Future[NodeSeq] = {
```

Comment:
I think you only use this inside fetchDepartments – you might as well just do this there.